src/Mod/Draft/draftobjects/dimension.py

```diff
@@ -353,29 +326,17 @@ def __init__(self, obj):
     def init_properties(self, obj):
         """Add Angular Dimension specific properties to the object and set them"""
 
-        obj.addProperty("App::PropertyAngle",
-                        "FirstAngle",
-                        "Draft",
-                        QT_TRANSLATE_NOOP("App::Property",
-                                          "Start angle of the dimension"))
-
-        obj.addProperty("App::PropertyAngle",
-                        "LastAngle",
-                        "Draft",
-                        QT_TRANSLATE_NOOP("App::Property",
-                                          "End angle of the dimension"))
-
-        obj.addProperty("App::PropertyVectorDistance",
-                        "Center",
-                        "Draft",
-                        QT_TRANSLATE_NOOP("App::Property",
-                                          "The center point of this dimension"))
-
-        obj.addProperty("App::PropertyAngle",
-                        "Angle",
-                        "Draft",
-                        QT_TRANSLATE_NOOP("App::Property",
-                                          "The measurement of this dimension"))
+        _tip = QT_TRANSLATE_NOOP("App::Property","Start angle of the dimension")
+        obj.addProperty("App::PropertyAngle", "FirstAngle", "Draft", )
+
+        _tip = QT_TRANSLATE_NOOP("App::Property","End angle of the dimension")
+        obj.addProperty("App::PropertyAngle", "LastAngle", "Draft", _tip)
+
+        _tip = QT_TRANSLATE_NOOP("App::Property", "The center point of this dimension")
+        obj.addProperty("App::PropertyVectorDistance", "Center", "Draft", _tip)
+
+        _tip = QT_TRANSLATE_NOOP("App::Property", "The measurement of this dimension")
+        obj.addProperty("App::PropertyAngle", "Angle", "Draft", _tip)
```

Comment:
I'm trying to add the properties with each argument per line, so that it is visually clear the type, the name, the group, and the tooltip. So, I wouldn't like to write it in a single line.

Reply:
I'd prefer it a little bit more compact... removing the tooltip from the addProperty function makes really easy to read the arguments...
Else we have 4 lines with just 4-5 characters each, it is a bit of a pain to me to scroll them when we have 10 properties added. (and we are still within 80 characters row)

Reply:
What I've been doing is grouping the properties in a function. And then calling the functions in a main `set_properties()` function. I think this helps find the function that we want better than if all properties are defined together. See `draftobjects/array.py`, for example.
```py
def set_properties(self, obj)
    self.set_main_properties(obj)
    self.set_small_properties(obj)
    self.set_big_properties(obj)

def set_main_properties(self, obj):
    _tip = _NOOP("Text")
    obj.addProperty(App::Something,
                    "Name",
                    "Group",
                    _tip)
```

Reply:
yes, having a set_property method seems good. Also Arch have it for objects.
It's out of the scope of this PR, but we can do it later!